src/products/views/ProductCreate.tsx

```diff
@@ -53,118 +52,111 @@ export const ProductCreateView: React.FC = () => {
-                warehouse: stock.id
-              })),
-              trackInventory: formData.trackInventory,
-              weight: weight(formData.weight)
-            }
-          });
-        };
-
-        return (
-          <>
-            <WindowTitle
-              title={intl.formatMessage({
-                defaultMessage: "Create Product",
-                description: "window title"
-              })}
-            />
-            <ProductCreatePage
-              currency={maybe(() => shop.defaultCurrency)}
-              categories={maybe(
-                () => searchCategoryOpts.data.search.edges,
-                []
-              ).map(edge => edge.node)}
-              collections={maybe(
-                () => searchCollectionOpts.data.search.edges,
-                []
-              ).map(edge => edge.node)}
-              disabled={productCreateOpts.loading}
-              errors={productCreateOpts.data?.productCreate.errors || []}
-              fetchCategories={searchCategory}
-              fetchCollections={searchCollection}
-              fetchProductTypes={searchProductTypes}
-              header={intl.formatMessage({
-                defaultMessage: "New Product",
-                description: "page header"
-              })}
-              productTypes={maybe(() =>
-                searchProductTypesOpts.data.search.edges.map(edge => edge.node)
-              )}
-              onBack={handleBack}
-              onSubmit={handleSubmit}
-              saveButtonBarState={productCreateOpts.status}
-              fetchMoreCategories={{
-                hasMore: maybe(
-                  () => searchCategoryOpts.data.search.pageInfo.hasNextPage
-                ),
-                loading: searchCategoryOpts.loading,
-                onFetchMore: loadMoreCategories
-              }}
-              fetchMoreCollections={{
-                hasMore: maybe(
-                  () => searchCollectionOpts.data.search.pageInfo.hasNextPage
-                ),
-                loading: searchCollectionOpts.loading,
-                onFetchMore: loadMoreCollections
-              }}
-              fetchMoreProductTypes={{
-                hasMore: maybe(
-                  () => searchProductTypesOpts.data.search.pageInfo.hasNextPage
-                ),
-                loading: searchProductTypesOpts.loading,
-                onFetchMore: loadMoreProductTypes
-              }}
-              warehouses={
-                warehouses.data?.warehouses.edges.map(edge => edge.node) || []
-              }
-              weightUnit={shop?.defaultWeightUnit}
-            />
-          </>
-        );
-      }}
-    </TypedProductCreateMutation>
+    <>
+      <WindowTitle
+        title={intl.formatMessage({
+          defaultMessage: "Create Product",
+          description: "window title"
+        })}
+      />
+      <ProductCreatePage
+        currency={maybe(() => shop.defaultCurrency)}
```

Comment:
we can remove maybe

Reply:
More like `shop?.defaultCurrency`,  but yeah